### DISCUSSION TITLE
Legal structure 🤔

### DISCUSSION THREAD

**Discussion by DanM3rcurius** [SecretDecks/Documentation#65] [Ideas]
### (charitable) Association/Verein (NGO) - 
- for international written & digital educational/spiritual (mental health content) content, and/or artist onboarding and incubation. 
- In Germany a charitable “Verein” (association) for would exempt us from taxes, since we would check many of the boxes (educational, health, arts, culture, …)
- cost to create a verein in Berlin: around 250€ (75€ fee + notary) ; 2 members for creation, 7 to have it listed
- @wlstylr do you by chance know how easy it is (and how much does it cost) to form a charitable association in the US and how well can we operate internationally? 

### LLC (via [Otoco.io](https://otoco.io/)) [*limited liability wrapper*] 
- for issuing or buying tokens, banking, cooperative fiat on and off ramp, contracting external service providers; 
- supplemented with golden share model! (either golden share goes to Purpose Stiftung (possible?!) or Catalyst!
- an LLC wrapper would give the DisCo an interface into the legacy financial system and economy - do we need that?

Thoughts @SecretDecks/founding-core ?

**Comment by stephen-rowan** [comment]
I am mulling over similar funding and community options for the Swarm GitHub Repos (See "GitHub Sponsors and other funding sources" https://github.com/Catalyst-Swarm/Catalyst-Beehive/issues/16 ). Looking particularly at https://opencollective.com/.

**Reply by wlstylr** [reply]
Open Collective looks like a promising alternative to a traditional bank account if you went the route of obtaining a fiscal host - https://docs.opencollective.com/help/fiscal-hosts/fiscal-hosts

In an ideal world we would be able to find a decentralized crypto / fiat onramp banking and payroll solution.

**Reply by DanM3rcurius** [reply]
with open collective , how well can we integrate with the token economy? 
I view it as an alternative to the charitable association/verein format. 
That could function as a stewardship layer, with an Otoco LLC as a economical layer (to issue tokens and for fiat on and off ramp for example).

**Reply by DanM3rcurius** [reply]
thanks for the open collective link @stephen-rowan 🙏

**Reply by stephen-rowan** [reply]
Yeah I was looking at open collective for the very specific use-case of accommodating GitHub Sponsors (https://github.com/sponsors). I think open-source projects that are developing Cardano governance models like Secret Decks and Swarm could attract corporate sponsors. Though it is fiat of course. 

I think there is a blockchain self-employment banking and payroll solution somewhere - but I forgot the company name ! In the current regulatory climate I guess it is about strategy - whether to use fiat financial platforms to build a community ?

**Comment by wlstylr** [comment]
> * @wlstylr do you by chance know how easy it is (and how much does it cost) to form a charitable association in the US and how well can we operate internationally?
> 

The first part is the same as filing through Otoco.io.  Once the LLC was complete we would then file for non-profit status by filing form 1023 with the IRS.  I don't believe there would be an additional cost associated with this unless we needed to hire an accountant or attorney to help - https://www.irs.gov/instructions/i1023

Just a gut feeling, but I feel like that fact that we are crypto related may raise some red flags with the IRS if we attempted to also file for non-profit status in the USA.   We would need an operating agreement, board of directors, and some language explicitly explains how we are a non-profit. 

The cost would run roughly anywhere from around $300 to $1200.

**Reply by DanM3rcurius** [reply]
ok so better approach is to segregate the crypto and collective layer? The question how it is in US is obsolete if we decide to go with Open Collective!

**Reply by wlstylr** [reply]
I think it's complicated.  Internally crypto and collective or community are tightly entwined.  

I don't see Open Collective as an answer to our legal structure.  To me it seems like it would solve the problem of allowing us to receive grants from organizations that operate in fiat, such as github, and track how those funds are dispersed.  It would provide a way for us to receive and manage fiat without actually having a corporate bank account. 

After receiving outside grants we would then most likely convert some to ADA or a stable coin which would be held in Secret Decks treasury, and then distributed in crypto to contributors, etc.  

Open Collective seems expensive in my opinion.  They take 10% plus payment processing fees. https://docs.google.com/document/u/1/d/e/2PACX-1vQbiyK2Fe0jLdh4vb9BfHY4bJ1LCo4Qvy0jg9P29ZkiC8y_vKJ_1fNgIbV0p6UdvbcT8Ql1gVto8bf9/pub
![Screenshot 2021-07-12 9 11 17 AM](https://user-images.githubusercontent.com/58446163/125321307-68e89200-e2f1-11eb-9fb0-e7f74382de4f.png)

Overall, I think open collective could help with allowing us to receiving and manage fiat grants in the short term.  In the USA it can be difficult to get approval for a corporate bank account as a crypto organization, and we will need a way to receive fiat and pay organizational costs as we scale.  

Another option would be to go with something like - https://www.kraken.com/en-us/bank

**Reply by DanM3rcurius** [reply]
> https://docs.google.com/document/u/1/d/e/2PACX-1vQbiyK2Fe0jLdh4vb9BfHY4bJ1LCo4Qvy0jg9P29ZkiC8y_vKJ_1fNgIbV0p6UdvbcT8Ql1gVto8bf9/pub

The fees only apply for fiscal hosts. Since we'd couple the open collective to the Otoco LLC and use this as opposed a fiscal host from OC.

**Reply by DanM3rcurius** [reply]
Otoco mentioned [Mercury](https://mercury.com/) as a good neo bank solution that works well with Otoco launched LLCs

**Comment by DanM3rcurius** [comment]
I just now  inquired with Purpose AG on what entity format to best choose.
